minard/polling.py

```diff
@@ -27,6 +26,43 @@ def polling_runs():
     return cmos_runs, base_runs
 
 
+def polling_history(crate, slot, channel):
+    '''
+    Return a list of form [[run number, cmos rate]]
+    for all runs with cmos data polling 
+    '''
+
+    conn = engine.connect()
+
+    result = conn.execute("SELECT run, cmos_rate from cmos WHERE crate = %i \
```

Comment:
Not super critical, but it's better to use %s and then pass the crate, card, channel as a tuple. See here: http://initd.org/psycopg/docs/usage.html#passing-parameters-to-sql-queries.